Building GUI on raspbian as per instructions does not work

**What is wrong**
Fresh Raspbian system. I'm following the canonical instructions for building the GUI as described here:
https://github.com/RespiraWorks/VentilatorSoftware/blob/master/gui/README.md
Installed all the qt packages by copypasting from the same readme.
It does not build.

**How do you know it is wrong**
Not building = bad.
Good would be if it builds. This used to work.
Since we are still unsure of how performant and reliable our docker-based deployment is, we have to be able to build it manually.

**Code pointers**
Possibly new dependencies have been added.

Here is the error I get:
```
./gui.sh --build
++ dirname ./gui.sh
+ cd .
++ uname -s
+ PLATFORM=Linux
+ '[' Linux '!=' Darwin ']'
+ '[' Linux '!=' Linux ']'
+ '[' --build == --help ']'
+ '[' --build == --install ']'
+ '[' --build == --build ']'
+ create_clean_directory build
+ dir_name=build
+ '[' -d build ']'
+ '[' -f build ']'
+ mkdir build
+ echo 'Clean directory created: build'
Clean directory created: build
+ return 0
+ qmake -unset QMAKEFEATURES
+ git submodule update --init --recursive
Submodule 'gui/app/third_party/qnanopainter' (https://github.com/QUItCoding/qnanopainter) registered for path 'src/third_party/qnanopainter'
Cloning into '/home/pi/VentilatorSoftware/gui/src/third_party/qnanopainter'...
Submodule path 'src/third_party/qnanopainter': checked out '8a74ead156ec81f27035c0f60d509cb93df6d2ce'
+ cd build
+ qmake ..
Info: creating stash file /home/pi/VentilatorSoftware/gui/build/.qmake.stash
+ make -j
cd src/ && ( test -e Makefile || /usr/lib/qt5/bin/qmake -o Makefile /home/pi/VentilatorSoftware/gui/src/src.pro ) && make -f Makefile 
Project MESSAGE: QNanoPainter Release build
Project MESSAGE: Including QNanoPainter OpenGL ES backends
Project ERROR: Unknown module(s) in QT: serialport
make: *** [Makefile:47: sub-src-make_first-ordered] Error 3
+ '[' --build == --test ']'
```

--- a/gui/README.md
+++ b/gui/README.md
@@ -9,7 +9,7 @@ It is based on QT.  **Please note** that on the Rpi, we are currently using vers
 To install the necessary Qt dependencies, use:
 
 ```
-sudo apt install build-essential qml-module-qtquick-controls2 qml-module-qtquick-layouts qtdeclarative5-dev qtquickcontrols2-5-dev libqt5quick* libqt5charts5-dev qt5-qmake qml-module-qtquick2 qt5-default qml-module-qtcharts
+sudo apt install build-essential qml-module-qtquick-controls2 qml-module-qtquick-layouts qtdeclarative5-dev qtquickcontrols2-5-dev libqt5quick*  libqt5serialport5-dev libqt5serialport5 qt5-qmake qml-module-qtquick2 qt5-default qml-module-qtcharts
 ```
 
 (TODO update these instructions to use 5.11.3)

--- a/gui/app/app.pro
+++ b/gui/app/app.pro
@@ -8,7 +8,7 @@ include( ../defaults.pri )
     error( "Couldn't find the libqnanopainter file!" )
 }
 
-QT += core quick charts serialport
+QT += core quick serialport
 
 TEMPLATE = app
 TARGET = ProjectVentilatorGUI

--- a/gui/gui.sh
+++ b/gui/gui.sh
@@ -73,8 +73,6 @@ if [ "$1" == "--install" ] ; then
     qtbase5-dev-tools \
     qtdeclarative5-dev \
     qtmultimedia5-dev \
-    libqt5charts5 \
-    libqt5charts5-dev \
     qml-module-qtcharts \
     libqt5multimedia5-plugins \
     qtquickcontrols2-5-dev \

--- a/gui/src/src.pro
+++ b/gui/src/src.pro
@@ -1,4 +1,4 @@
-QT += core quick charts serialport qml
+QT += core quick serialport qml
 
 include( ../defaults.pri )
 ! include( ../common.pri ) {

